geomstats/geometry/product_riemannian_metric.py

```diff
@@ -33,10 +33,9 @@ def __init__(self, metrics, default_point_type='vector', n_jobs=1):
 
         sig_0 = sum(sig[0] for sig in signatures)
         sig_1 = sum(sig[1] for sig in signatures)
-        sig_2 = sum(sig[2] for sig in signatures)
         super(ProductRiemannianMetric, self).__init__(
             dim=sum(dims),
-            signature=(sig_0, sig_1, sig_2),
+            signature=(sig_0, sig_1),
```

Comment:
NIT: instead of sig_0, sig_1 we should rename this into `sig_pos` and `sig_neg` (or `sig_neg` and sig_pos`, I forget the order) for clarity.

Reply:
done